### PR TITLE
delete unnecessary parameter

### PR DESCRIPTION
**Pull request by realwanpengli**
for https://github.com/Azure/azure-signalr/pull/534#issuecomment-493949170

### CODE REVIEW

File: src/Microsoft.Azure.SignalR.Management/ServiceManager.cs [hunk 1]
```diff
@@ -30,7 +31,13 @@ internal ServiceManager(ServiceManagerOptions serviceManagerOptions, string prod
         {
             _serviceManagerOptions = serviceManagerOptions;
             _endpoint = new ServiceEndpoint(_serviceManagerOptions.ConnectionString, EndpointType.Secondary);
-            _endpointProvider = new ServiceEndpointProvider(_endpoint, appName: _serviceManagerOptions.ApplicationName);
+            var optionWrapper = new OptionsWrapper<ServiceOptions>(new ServiceOptions
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Management/ServiceManager.cs]: use `Options.Create`

File: src/Microsoft.Azure.SignalR/EndpointProvider/ServiceEndpointProvider.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ internal class ServiceEndpointProvider : IServiceEndpointProvider
         private readonly TimeSpan _accessTokenLifetime;
         private readonly IServiceEndpointGenerator _generator;
 
-        public ServiceEndpointProvider(ServiceEndpoint endpoint, string appName = "", IOptions<ServiceOptions> serviceOptions = null, TimeSpan? ttl = null)
+        public ServiceEndpointProvider(ServiceEndpoint endpoint, IOptions<ServiceOptions> serviceOptions, TimeSpan? ttl = null)
         {
             var connectionString = endpoint.ConnectionString;
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR/EndpointProvider/ServiceEndpointProvider.cs]: for `serviceOptions`, check null

Comment (realwanpengli) [reply src/Microsoft.Azure.SignalR/EndpointProvider/ServiceEndpointProvider.cs]: But we alreadly check `connectionString` in next line (line 28)

Comment (realwanpengli) [reply src/Microsoft.Azure.SignalR/EndpointProvider/ServiceEndpointProvider.cs]: I think passing serviceOption is unnecessary, the provider uses ApplicationName only

Comment (realwanpengli) [reply src/Microsoft.Azure.SignalR/EndpointProvider/ServiceEndpointProvider.cs]: already update in commit a3513f7

**Pull request closed without merging**